### PR TITLE
Remove extra derivative overloads

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
I think these have all been upstreamed by now. Tests would tell us.

### CODE REVIEW

File: src/DiffEqFlux.jl [hunk 1]
```diff
@@ -4,7 +4,7 @@ using Adapt, Base.Iterators, ConsoleProgressMonitor, DataInterpolations,
     DiffEqBase, Distributions, DistributionsAD,
     ForwardDiff, LinearAlgebra, Lux,
     Logging, LoggingExtras, Printf, ProgressLogging, Random, RecursiveArrayTools,
-    Reexport, SciMLBase, TerminalLoggers, Zygote, ZygoteRules
+    Reexport, SciMLBase, TerminalLoggers
```
Comment (ChrisRackauckas) [inline src/DiffEqFlux.jl:7]:
```suggestion
    Reexport, SciMLBase, TerminalLoggers, Zygote
```

**Comment by prbzrg** [comment]
I think `ForwardDiff` can also be removed.

**Comment by ChrisRackauckas** [comment]
Needs https://github.com/FluxML/Zygote.jl/pull/1362

**Comment by ChrisRackauckas** [comment]
old

**Pull request closed without merging**